lib/ui/puyo/game-duel.js

```diff
@@ -40,6 +40,17 @@ export default class PuyoDuelGame extends PuyoGameBase {
     } else {
       this.playerGrid.installEventListeners();
     }
+
+    window.setInterval(() => {
```

Comment:
Made a separate issue for that. https://github.com/frostburn/panel-league/issues/154